smoketests/tests/servers.py

```diff
@@ -7,19 +7,31 @@ class Servers(Smoketest):
     def test_servers(self):
         """Verify that we can add and list server configurations"""
 
-        out = self.spacetime("server", "add", "https://testnet.spacetimedb.com", "testnet", "--no-fingerprint")
+        out = self.spacetime("server", "add", "--url", "https://testnet.spacetimedb.com", "testnet", "--no-fingerprint")
         self.assertEqual(extract_field(out, "Host:"), "testnet.spacetimedb.com")
         self.assertEqual(extract_field(out, "Protocol:"), "https")
 
         servers = self.spacetime("server", "list")
         self.assertRegex(servers, re.compile(r"^\s*testnet\.spacetimedb\.com\s+https\s+testnet\s*$", re.M))
         self.assertRegex(servers, re.compile(r"^\s*\*\*\*\s+127\.0\.0\.1:3000\s+http\s+localhost\s*$", re.M))
 
-        out = self.spacetime("server", "fingerprint", "-s", "http://127.0.0.1:3000", "-y")
+        out = self.spacetime("server", "fingerprint", "http://127.0.0.1:3000", "-y")
         self.assertIn("No saved fingerprint for server 127.0.0.1:3000.", out)
 
-        out = self.spacetime("server", "fingerprint", "-s", "http://127.0.0.1:3000")
+        out = self.spacetime("server", "fingerprint", "http://127.0.0.1:3000")
         self.assertIn("Fingerprint is unchanged for server 127.0.0.1:3000", out)
 
-        out = self.spacetime("server", "fingerprint", "-s", "localhost")
+        out = self.spacetime("server", "fingerprint", "localhost")
         self.assertIn("Fingerprint is unchanged for server localhost", out)
+
+    def edit_server(self):
+        """Verify that we can edit server configurations"""
+
+        out = self.spacetime("server", "add", "--url", "https://foo.com", "foo", "--no-fingerprint")
+        out = self.spacetime("server", "edit", "--url", "https://testnet.spacetimedb.com", "--new-name", "testnet", "--no-fingerprint")
```

Comment:
so it turns out that `edit` can implicitly use the default server - do we like this behavior?

Reply:
ah `ping` and `fingerprint` also implicitly use the default server. I kinda feel like all of these commands should take an explicit server?

Reply:
I feel like explicit is probably the way to go here. I don't think the implicit assumption adds a lot to the user experience and we can always add it back later if we want. (can't remove it though).